include/dlaf/eigensolver/bt_band_to_tridiag.h

```diff
@@ -90,6 +89,6 @@ void backTransformationBandToTridiag(comm::CommunicatorGrid grid, const SizeType
   DLAF_ASSERT(band_size >= 2, band_size);
   DLAF_ASSERT(mat_hh.blockSize().rows() % band_size == 0, mat_hh.blockSize(), band_size);
 
-  internal::BackTransformationT2B<B, D, T>::call(grid, band_size, mat_e, mat_hh);
+  BackTransformationT2B<B, D, T>::call(grid, band_size, mat_e, mat_hh);
```

Comment:
In addition, the non-BT version is `BandToTridiag`. Not worth too many cycles since it's internal, but probably some simple renaming to make them a bit more consistent might be worth it.

Reply:
I would leave this for the next iteration. The main target of this PR is to have a reasonable naming for public interface.